pkg/protocol/uri_windows.go

```diff
@@ -53,3 +55,24 @@ func addLeadingSlash(dst, src []byte) []byte {
 
 	return dst
 }
+
+// checkSchemeWhenCharIsColon check url begin with :
+// Scenarios that handle protocols like "http:"
+// Add the path to the win file, e.g. "E:/gopath", "E:/".
+func checkSchemeWhenCharIsColon(i int, rawURL []byte) (scheme, path []byte) {
+	if i == 0 {
+		hlog.Errorf("error happened when trying to parse the rawURL(%s): missing protocol scheme", rawURL)
+		return
+	}
+	if i+2 < len(rawURL) && rawURL[i+1] == '/' {
+		if rawURL[i+2] == '/' {
+			return rawURL[:i], rawURL[i+1:]
+		}
+		return nil, rawURL
+	}
+
+	if len(rawURL) == i+2 && rawURL[i+1] == '/' {
```

Comment:
And the case that `rawURL < i+2`like `http:`?
Make sure to align with the action in UNIX.